fe/fe-core/src/main/java/org/apache/doris/common/proc/BackendsProcDir.java

```diff
@@ -184,6 +204,80 @@ public static List<List<String>> getBackendInfos() {
         return backendInfos;
     }
 
+    /**
+     * get backends disk info
+     *
+     * @return
+     */
+    public static List<List<String>> getBackendsDiskInfos() {
+        final SystemInfoService systemInfoService = Env.getCurrentSystemInfo();
+        List<List<String>> backendsDiskInfos = new LinkedList<>();
+        List<Long> backendIds = systemInfoService.getAllBackendIds(false);
+        if (backendIds == null) {
+            return backendsDiskInfos;
+        }
+
+        List<List<Comparable>> comparableBackendsDiskInfos = new LinkedList<>();
+        for (long backendId : backendIds) {
+            Backend backend = systemInfoService.getBackend(backendId);
+            if (backend == null) {
+                continue;
+            }
+
+            ImmutableMap<String, DiskInfo> disksRef = backend.getDisks();
+            for (DiskInfo disk : disksRef.values()) {
+                if (disk.getState() == DiskState.ONLINE) {
```

Comment:
May be we should also show the offline disk. The offline disk is also reported from be to master. (Offline disk means the disk health check is bad). And we should add a column to show this disk's online/offline state.

Reply:
ok